### PR TITLE
Task/print incoming request headers

### PR DESCRIPTION
**Pull request by dmoranj**
Address issue #64 

Adds a reques trace in debug mode to print the headers and body or the requesxt.

### CODE REVIEW

File: lib/fiware-orion-pep.js [hunk 1]
```diff
@@ -126,6 +126,21 @@ function createDynamicMiddlewareExecutor(proxyObj) {
     };
 }
 
+function traceRequest(req, res, next) {
+    logger.debug('Request for path [%s] from [%s]', req.path, req.get('host'));
+    logger.debug('Headers:\n%j\n', req.headers);
+
+    if (req.headers['content-type'] === 'application/json') {
+        logger.debug('Body:\n\n%s\n\n', JSON.stringify(req.body, null, 4));
```
Comment (fgalan) [inline lib/fiware-orion-pep.js]: Should be '%f'? As you did in your previous PR.

Comment (dmoranj) [inline lib/fiware-orion-pep.js]: Not exactly the same case, as this middleware isn't even loaded when the log level is not DEBUG... I'm not sure between changing it for coherence or leaving it as it is for ease of debugging.

Comment (dmoranj) [inline lib/fiware-orion-pep.js]: Fixed in 04b1c8f (completeing the doc).

**Comment by fgalan** [comment]
LGTM

**Pull request merged**